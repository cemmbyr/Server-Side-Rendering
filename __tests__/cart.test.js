/** @jest-environment jsdom */
import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Cart from '../client/components/cart';

it('sum text correctly', () => {
  const { getByTestId } = render(<Cart />);

  expect(getByTestId('sum')).toHaveTextContent('Toplam Tutar: 0');
});

it('snapshot', () => {
  const dom = renderer.create(<Cart />).toJSON();

  expect(dom).toMatchSnapshot();
});
