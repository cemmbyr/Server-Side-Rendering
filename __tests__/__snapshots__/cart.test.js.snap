// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="cart"
>
  <div
    className="cart-sum"
    data-testid="sum"
  >
    Toplam Tutar: 
    0
  </div>
</div>
`;
